catalogue/webapp/components/CalendarInput/CalendarInput.tsx

```diff
@@ -0,0 +1,101 @@
+import { forwardRef, useContext, useState } from 'react';
+// $FlowFixMe (tsx)
```

Comment:
We're in a TS file, aren't we?(!)

Reply:
we are! - copy and pasted, should have noticed